charts/common/templates/_version.tpl

```diff
@@ -0,0 +1,41 @@
+{{/*
+Version of application to be deployed
+*/}}
+{{- define "accelleran.common.appVersion" -}}
+{{- $ := get . "top" | required "The top context needs to be provided to common app version" -}}
+{{- $values := get . "values" | default $.Values -}}
+
+{{- $version := ($values.image).tag | default $.Values.global.tag | default $.Chart.AppVersion -}}
+
+{{- if not $version -}}
+{{- /*
+Work-around as .Chart.AppVersion is undefined with (de)serializations (from/toYaml)
+This can not be used with `default` as then it would be evaluated,
+resulting in a failure when no (de)serializations took place.
```

Comment:
Is this a "known" issue? Can we link to a Github ticket for it?

Reply:
Ah, I guess it's caused by [this](https://github.com/helm/helm/blob/db4d20f3ad3ebbd8d17926e7a3cc7da7c177b52b/pkg/chart/metadata.go#L68C49-L68C49).. 🤔

Reply:
Don't remember anything about it to be honest. It's even not coming back with the comment...